src/main/views/courts/tabs/typesContent.njk

```diff
@@ -0,0 +1,138 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+<h2 class="govuk-heading-l">Court Types and Codes</h2>
+
+{% if errorMsg != '' %}
+  {{ govukErrorSummary({
+    titleText: "There is a problem",
+    errorList: [
+      {
+        text: errorMsg
+      }
+    ]
+  }) }}
+{% endif %}
+
+{% if updated === true %}
+  {{ govukPanel({
+    titleText: "Court Types updated"
+  }) }}
+{% endif %}
+
+
+{% for item in items %}
+
+{% if item.text === "Magistrates' Court"  %}
+
+ {% set magistratesCodeError = false %}
+  {% if item.checked and item.code | valid === false %}
+   {% set magistratesCodeError = { text: 'Magistrates code is required and must be numeric.' } %}
```

Comment:
If the magistrates code must be numeric, does the govUkInput below need the same validation as the others (inputmode and pattern set etc)?

Reply:
added input mode and pattern